### PR TITLE
Add back Tika component

### PR DESCRIPTION
**Pull request by zhfeng**
It could close
- #4470 
- #3599

### CODE REVIEW

**Review by zbendhiba — APPROVED** [review]

**Review by oscerd — APPROVED** [review]

**Comment by zhfeng** [comment]
@jamesnetherton It seems the maven connection timeout happens again and again. I just find
- https://github.com/actions/runner-images/issues/1499
and mentioned a work around to add 
`-Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false -Dmaven.wagon.httpconnectionManager.ttlSeconds=120`.

How do you think about it?

**Comment by jamesnetherton** [comment]
> How do you think about it?

We can add it to `CQ_MAVEN_ARGS` in each of our workflow YAML files.

File: poms/bom/pom.xml [hunk 1]
```diff
@@ -6266,6 +6266,17 @@
                 <groupId>io.quarkiverse.tika</groupId>
                 <artifactId>quarkus-tika</artifactId>
                 <version>${quarkiverse-tika.version}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>commons-logging</groupId>
+                        <artifactId>commons-logging</artifactId>
+                    </exclusion>
+                    <!-- It could be removed when https://github.com/quarkiverse/quarkus-tika/issues/109 gets resolved -->
```
Comment (jamesnetherton) [inline poms/bom/pom.xml:6274]: Unless we can get a quick release of `quarkus-tika` containing the fix for this, we should probably open a follow-up issue here as a reminder to remove this exclusion.

**Review by jamesnetherton — APPROVED** [review]

File: poms/bom/pom.xml (hunk 1 above)
Comment (zhfeng) [reply poms/bom/pom.xml:6274]: I will take a new releas of `quarkus-tika` ASAP.

Comment (zhfeng) [reply poms/bom/pom.xml]: It should be fine with `2.0.0.CR2`.

**Review by jamesnetherton — APPROVED** [review]

**Review by aldettinger — APPROVED** [review]

**Pull request merged**